aws-postgresql.yml

```diff
@@ -23,33 +23,6 @@ support_url: https://aws.amazon.com/rds/postgresql/
 tags: [aws, postgresql, postgres, beta]
 plan_updateable: true
 plans:
-- name: small
-  id: ffc51616-228b-41bd-bed1-d601c18d58f5
-  description: 'Beta - PostgreSQL 11, minimum 2 cores, minimum 4GB ram, 5GB storage'
-  display_name: "small (Beta)"
-  properties:
-    cores: 2
-    storage_gb: 5
-    postgres_version: 11
-    subsume: false
-- name: medium
-  id: e64d07f9-ceb2-40a6-abd9-391047fa3cf5
-  description: 'Beta - PostgreSQL 11, minimum 4 cores, minimum 8GB ram, 10GB storage'
-  display_name: "medium (Beta)"
-  properties:
-    cores: 4
-    storage_gb: 10
-    postgres_version: 11
-    subsume: false
-- name: large
-  id: 48baef10-a14c-4ae1-aab5-25f26eba941a
-  description: 'Beta - PostgreSQL 11, minimum 8 cores, minimum 16GB ram, 20GB storage'
-  display_name: "large (Beta)"
-  properties:
-    cores: 8
-    storage_gb: 20
-    postgres_version: 11
-    subsume: false
 - name: subsume
```

Comment:
question (blocking): is there any particular reason to leave the subsume plan?

Reply:
My thinking here is that subsume plan is the special plan that we offer to be able to onboard existing instances onto the broker. This is not a "normal" plan that an operator would want to create/modify so exposing it in the tile felt like it might be confusing. 

There is an ongoing conversation about the future of subsume - and by keeping it in the brokerpak we can keep control of what we do with it (modify/completely remove it etc)

Reply:
Thank you very much for replying to me. Great information!